db/migrate/20220304145225_create_places.rb

```diff
@@ -0,0 +1,33 @@
+class CreatePlaces < ActiveRecord::Migration[6.1]
+  def change
+    create_table :places do |t|
+      t.string :name, limit: 100, null: false
+      t.integer :type
+      t.string :city
+      t.string :region
+      t.integer :rooms_available
+      t.integer :beds
+      t.integer :kids_beds
+      t.boolean :is_pets_allowed
+      t.integer :capacity, default: 2, null: false
+      t.integer :additional_child_place, default: 0
+      t.integer :coordinator
```

Comment:
sorry, I didn't noticed that, but @s-nestor noticed, it'd be better to have coordinator_id

Reply:
Done
Addressed here https://github.com/vova/war_shelter/pull/22/commits/7f8f34ffb8954478c8913c7b279be9aa00b3c4ef